Translation/en/Text/Action_Lists/zz_MachineTranslation.txt

```diff
@@ -118,7 +118,7 @@ TVを見るのをやめる=Stop watching TV
 // act_filter_f_11[00_00] (column 2)
 
 //ごろごろする=roll about (e.g. with laughter)
-ボーッとする=be in a daze
+//ボーッとする=be in a daze
```

Comment:
It seems that you forgot to comment lines past this one. I suggest the tool " MTL Duplicates": https://github.com/SpockBauru/TranslationTools_Illusion#mtl-duplicates